src/i18n/en.js

```diff
@@ -321,6 +321,7 @@ deletion, security, cross-border data transfers and other issues.
     permission_description: 'Custom text',
     permission_name: 'Field Name',
     permission_value: 'Value',
+    tab_description: 'Subscriber GDPR state',
```

Comment:
Have you checked this with @mrunge86 ?

Reply:
No I didn't review the content yet, I'm reusing what we already had.

Reply:
What do you think about reusing _dapeg_description_?